src/main/java/website/frontrow/board/Mover.java

```diff
@@ -149,10 +150,11 @@ public void tick(Level level)
         this.newMotion = new Point(0, 0);
 
         double x = this.motion.getX();
-        this.motion.setX(
-                Math.max(Math.min(x, GameConstants.MAX_X_SPEED),
-                        -GameConstants.MAX_X_SPEED));
-
+        
+    	this.motion.setX(
+    			Math.max(Math.min(x, GameConstants.MAX_X_SPEED),
+    					-GameConstants.MAX_X_SPEED) * this.getUnitSpeedMultiplier());
```

Comment:
It's more understandable this way. Without indentation, it would be worse.

< layout